.agents/skills/gstack/plan-eng-review/SKILL.md

```
@@ -1098,7 +1098,7 @@ At the end of the review, fill in and display this summary so the user can see a
 - Performance Review: ___ issues found
 - NOT in scope: written
 - What already exists: written
-- TODOS.md updates: ___ items proposed to user
+- Linear follow-ups: ___ issues created / ___ proposed to user
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)